draft-demarco-oauth-status-attestations.md

```diff
@@ -63,24 +63,24 @@ Status Assertions ensure the non-revocation of digital
 credentials, whether in JSON Web Tokens (JWT) or CBOR Web Tokens (CWT)
 format. Status Assertions function
 similarly to OCSP Stapling, allowing wallet instances to present
-time-stamped assertions from the Credential Issuer.
+time-stamped assertions provided by the Credential Issuer.
 The approach outlined in this specification enables the
 verification of credentials against revocation without
-direct queries to third-party systems.
-This enhances privacy, reduces latency, and
-facilitates offline verification.
+direct queries to third-party systems,
+enhancing privacy, reducing latency, and
+faciliting offline verification.
 
 The figure below illustrates the process by which a Wallet Instance
 requests and obtains a Status Assertion from the credential issuer.
 
 ~~~ ascii-art
-+-----------------+                             +-------------------+
-|                 | Requests Status Assertion   |                   |
-|                 |---------------------------->|                   |
-| Wallet Instance |                             | Credential Issuer |
-|                 | Status Assertion            |                   |
-|                 |<----------------------------|                   |
-+-----------------+                             +-------------------+
++-----------------+                              +-------------------+
+|                 | Requests Status Assertions   |                   |
+|                 |----------------------------->|                   |
+| Wallet Instance |                              | Credential Issuer |
+|                 | Status Assertions            |                   |
+|                 |<-----------------------------|                   |
++-----------------+                              +-------------------+
```

Comment:
I think it would help if we have a common base terminology between status list and status attestations, as we are adopted by OAuth WG, we've used Client instead of Wallet and Referenced Token instead of Credential. Happy to negotiate :)

Reply:
I agree with this comment, we should use the term client... and explain that wallet's are a common type of client... Ideally we could all share a reference for wallet that disambiguated, hardware, mobile, cloud and virtual wallets.

Reply:
Client wins, thank you for this.

Consider It done